tests/queries/models.py

```diff
@@ -613,13 +613,14 @@ def __str__(self):
 
 
 class BaseUser(models.Model):
-    pass
+    annotation = models.ForeignKey(Annotation, models.CASCADE, null=True, blank=True, related_name="base_users")
 
 
 class Task(models.Model):
     title = models.CharField(max_length=10)
-    owner = models.ForeignKey(BaseUser, models.CASCADE, related_name='owner')
-    creator = models.ForeignKey(BaseUser, models.CASCADE, related_name='creator')
+    owner = models.ForeignKey(BaseUser, models.CASCADE, related_name='tasks')
+    creator = models.ForeignKey(BaseUser, models.CASCADE, related_name='other_tasks')
```

Comment:
Do we need to change `related_name` here?  We could add `note` with `related_name='owner'` instead.